test/inductor/test_aot_inductor.py

```diff
@@ -1162,6 +1162,21 @@ def forward(self, q, k, v):
         aot_model = torch._export.aot_load(path, device=self.device)
         torch.testing.assert_close(m(*inputs), aot_model(*inputs))
 
+    def test_aoti_constant_tensor(self):
+        class Foo(torch.nn.Module):
+            def __init__(self):
```

Comment:
You need to specify a device here to cover both cpu and cuda, like here https://github.com/pytorch/pytorch/blob/a41413829c98377e4c155ff150f250438303f7b2/test/inductor/test_aot_inductor.py#L210 .